src/Cake.Testing.Xunit/WindowsFact.cs

```diff
@@ -8,17 +8,17 @@
 
 namespace Cake.Testing.Xunit
 {
-    public sealed class WindowsFact : FactAttribute
+    public sealed class WindowsFactAttribute : FactAttribute
```

Comment:
This is the only change needed (+ rename file), compiler will add `Attribute` automatically,

Reply:
Thanks, done